llap-tez/src/java/org/apache/hadoop/hive/llap/tezplugins/metrics/DummyLlapTaskSchedulerMetrics.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.hadoop.hive.llap.tezplugins.metrics;
+
+import org.apache.hadoop.conf.Configuration;
+
+public class DummyLlapTaskSchedulerMetrics extends LlapTaskSchedulerMetrics {
+  public DummyLlapTaskSchedulerMetrics(Configuration conf) {
+    super(conf);
+  }
+}
```

Comment:
This class seems to have the same behavior as its parent `LlapTaskSchedulerMetrics` and therefore it is not a dummy metrics handler? I though the idea of `DummyLlapTaskSchedulerMetrics` was to implement metrics handling with dummy functionality.

Reply:
yeah, that's what cause so many testing failures, I'm working on it!
thanks for taking a look